### PR TITLE
Message sending timeouts

### PR DESCRIPTION
**Pull request by adamant-al**

### CODE REVIEW

**Review by yoxira — APPROVED** [review]
Good feature!

**Comment by bludnic** [comment]
Need to update the OpenAPI schema

File: assets/general/adamant/info.json [hunk 1]
```diff
@@ -24,6 +24,10 @@
     "oldPendingInterval": 4000,
     "registeredInterval": 4000
   },
+  "messages": {
+    "messageTimeout": 300000,
+    "filesTimeout": 300000
+  },
```
Comment (bludnic) [inline assets/general/adamant/info.json:30]: I suggest this structure:

```suggestion
  "timeout": {
    "message": 300000,
    "file": 300000
  },
```

Comment (yoxira) [reply assets/general/adamant/info.json:30]: what about `file` -> `attachment`?

Comment (bludnic) [reply assets/general/adamant/info.json:30]: @yoxira also good

@adamant-al My only concern is about `messages` naming. It seems like we want to describe an array of something, which is obviously not true.

Comment (adamant-al) [reply assets/general/adamant/info.json:30]: Accepted.
@bludnic Push the changes.

**Comment by adamant-al** [comment]
Great.

**Review by bludnic — APPROVED** [review]

**Pull request merged**